hooks/pre/10-reset_data.rb

```diff
@@ -56,18 +56,24 @@ def reset_candlepin
   empty_db_in_postgresql('candlepin')
 end
 
-def pg_command_base(config, command, args)
-  "PGPASSWORD='#{config[:password]}' #{command} -U #{config[:username]} -h #{config[:host]} -p #{config[:port]} #{args}"
+def pg_env(config)
+  {
+    'PGHOST' => config.fetch(:host, 'localhost'),
```

Comment:
Do we need a default or can we trust psql to figure it out?

Reply:
No idea, copied from your fm patch ;)

Reply:
There I had it because `local?` compares to `localhost` and I didn't want to change that. Perhaps we should make it more robust so it can parse any postgresql url. I'd like to be able to use unix sockets and ident auth but now we'd break foreman-maintain

Reply:
Here we have `remote_host?(hostname)` which does something very similar.

The old code just assumed `config[:host]` is set (the command would be wrong if it were unset), which we can do here too?

Reply:
I think keeping a default for localhost is OK for now then.